### PR TITLE
google tag manager script

### PR DESCRIPTION
**Pull request by DavidGeismarLtd**
gtm @ManyBaby maybe it's better as a javascript pack tag too, right ? + we are loading font awesome  via cdn   <script src="https://kit.fontawesome.com/d42c90e7ee.js" crossorigin="anonymous"></script>.
Do you know if it's a good idea or is it better to include it as a npm package ?

### CODE REVIEW

**Comment by ManyBaby** [comment]
I believe that it's better to centralize all the js code into the webpack pipeline so we can better optimize how are pack are loaded. As for the CDN, they are notoriously better than us at serving static assets and I think that as long as their uptime is good enough we shouldn't have to worry. In thi case icons are not critical and the website will work without them. But it leads me to another question. Rémi has provided SVG icons to my knowledge. Why aren't they sufficient @DavidGeismarLtd ?

**Comment by ManyBaby** [comment]
Sorry @DavidGeismarLtd  J'ai 💩  . J'ai poussé sur ta branche ma proposition de refacto alors que je voulais faire une autre branche. Dis moi si ça te convient sinon je rollback mes commits.

**Comment by DavidGeismarLtd** [comment]
> Sorry @DavidGeismarLtd J'ai 💩 . J'ai poussé sur ta branche ma proposition de refacto alors que je voulais faire une autre branche. Dis moi si ça te convient sinon je rollback mes commits.

all good pour moi, tu sais a quoi il sert le fichier noscript ?

File: app/views/shared/_google_tag_manager_noscript.html.erb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!-- Google Tag Manager (noscript) -->
+<noscript>
+  <iframe
+    src="https://www.googletagmanager.com/ns.html?id=GTM-KHRJ7HF"
+    height="0"
+    width="0"
```
Comment (DavidGeismarLtd) [inline app/views/shared/_google_tag_manager_noscript.html.erb:6]: Tu sais a quoi il sert ce fichier noscript @ManyBaby ?

Comment (ManyBaby) [reply app/views/shared/_google_tag_manager_noscript.html.erb:6]: Ce qui est à l'intérieur de cette balise s'active uniquement quand certaines personnes n'ont pas js activé sur leur navigateur.

**Pull request merged**